extensions/amp-story/1.0/_locales/en.js

```diff
@@ -310,6 +314,10 @@ const strings = {
       'Label for the sidebar button that pulls up a menu ' +
       'of options for interacting with the story',
   },
+  [LocalizedStringId.AMP_STORY_SKIP_NEXT_BUTTON_LABEL]: {
+    string: 'Skip next',
+    description: 'Label for a button that advances the story to the next page',
```

Comment:
Nit: worth adding context saying that it'll skip the current content to advance to the next element of a carousel. The story / page vocabulary is super confusing to people unfamiliar with AMP Stories